applications/DEMApplication/custom_strategies/strategies/explicit_solver_continuum.cpp

```diff
@@ -582,7 +582,7 @@ namespace Kratos {
             }//while
 
                 if (iteration < maxiteration){
-                KRATOS_INFO("DEM") << "The iterative procedure converged after " << iteration << " iterations, to value \e[1m" << current_coordination_number << "\e[0m using a global amplification of radius of " << amplification << ". " << "\n" << std::endl;
+                KRATOS_INFO("DEM") << "The iterative procedure converged after " << iteration << " iterations, to value " << current_coordination_number << " using a global amplification of radius of " << amplification << ". " << "\n" << std::endl;
```

Comment:
@maceligueta is this ok?
If you insist on keeping it then I think the easiest would be to use the macros to check where Kratos is compiled

Reply:
@maceligueta I just saw that you gave your thumbs up already, guess then it is ok

Reply:
Actually, I removed this 'font formatting' from a couple of places more (long ago) and this one remained unintentionally.